integrated_channels/integrated_channel/transmitters/content_metadata.py

```diff
@@ -32,49 +32,48 @@ def __init__(self, enterprise_configuration, client=IntegratedChannelApiClient):
             client=client
         )
 
-    def _log_info(self, msg, content_id=None):
+    def _log_info(self, msg, course_or_course_run_key=None):
         LOGGER.info(
             generate_formatted_log(
                 self.enterprise_configuration.channel_code(),
                 self.enterprise_configuration.enterprise_customer.uuid,
                 None,
-                content_id,
+                course_or_course_run_key,
                 msg
             )
         )
 
-    def _log_error(self, msg):
+    def _log_error(self, msg, course_or_course_run_key=None):
         LOGGER.error(
             generate_formatted_log(
                 self.enterprise_configuration.channel_code(),
                 self.enterprise_configuration.enterprise_customer.uuid,
                 None,
-                None,
+                course_or_course_run_key,
                 msg
             )
         )
 
-    def transmit(self, create_payload, update_payload, delete_payload, content_updated_mapping, **kwargs):
+    def _log_info_for_each_item_map(self, item_map, msg):
+        for content_id, transmission in item_map.items():
+            self._log_info(
+                f'integrated_channel_content_transmission_id={transmission.id}, '
+                f'{msg}',
+                course_or_course_run_key=content_id
+            )
+
+    def transmit(self, create_payload, update_payload, delete_payload, **kwargs):
         """
         Transmit content metadata items to the integrated channel. Save or update content metadata records according to
         the type of transmission.
         """
-        self._log_info(
-            f'Transmitting delete payload: {delete_payload} for customer: '
-            f'{self.enterprise_configuration.enterprise_customer.uuid}'
-        )
+        self._log_info_for_each_item_map(delete_payload, 'transmitting delete')
```

Comment:
these payloads can be large enough that they're cut-off/truncated in splunk which really diminishes their value - instead we should log a line per item using the helper above. 

later on we can work on tools to tune log verbosity, perhaps allowing changes on a per-customer or per-integration level while we're debugging a specific case while reducing the log load otherwise.